### ISSUE TITLE
openwrt package?

### ISSUE THREAD

**Issue by trungpham** [VrayoSystems/vtrunkd#230]
is it possible to create a package for openwrt routers?

**Comment by grandrew** [comment]
yes you can use the Makefile for openwrt - it is in the tree. Just add it to the wrt tree and the package is done. Note that it will require the source to be manually cloned into the directory openwrt_src (see makefile)

The project does not have any openwrt package maintainers  - so there is no official package.
